### PR TITLE
#204 publication fields

### PR DESCRIPTION
**Pull request by banukutlu**
Connected to #204 

- index and display publication fields
- to hide the publication statement label in search results added `psul_index_presenter.rb` and `_index_default.html.erb` partial (there might be a better way to solve this)
- also reordered the single item labels

### CODE REVIEW

File: spec/presenters/psul_index_presenter_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      end
+
+      let(:config) do
+        Blacklight::Configuration.new.configure do |config|
+          config.add_index_field 'publication_display_ssm'
+          config.add_index_field 'format'
+        end
+      end
+
+      it 'publication statement has an empty label' do
```
Comment (cdmo) [inline spec/presenters/psul_index_presenter_spec.rb]: Maybe this should read like `it 'should not have a label for publication statement, but still have a label for format' do` ?

**Comment by banukutlu** [comment]
> Maybe just a small change to the spec's language. Other then that it looks great. I can't think of a more elegant way to handle this honestly. And it might become a useful avenue for future front-end work and tweaks.
> 
> I feel like to do this "right" we'd have to do a significant amount of work in re-working BL itself. Like, a new conf setting could be added for "show_label" with a value of "true|false|record|results" or something like that and then add a method to create the html itself (i.e., `content_tag()`) based on that toggle.

💯

File: spec/presenters/psul_index_presenter_spec.rb (hunk 1 above)
Comment (banukutlu) [reply spec/presenters/psul_index_presenter_spec.rb]: sure! I just changed it but had to leave `should` out because rubocop was not happy with it.

**Comment by cdmo** [comment]
wonder why you merged in master too?

**Comment by banukutlu** [comment]
> wonder why you merged in master too?

there were conflicts, I used GH's Resolve Conflicts then commit

File: spec/presenters/psul_index_presenter_spec.rb (hunk 1 above)
Comment (cdmo) [reply spec/presenters/psul_index_presenter_spec.rb]: will have to think about that one!

**Comment by cdmo** [comment]
> > wonder why you merged in master too?
> 
> there were conflicts, I used GH's Resolve Conflicts then commit

Ah ok, interesting.

I've been loving rebase lately. But, rebasing here would mean a new PR.

File: spec/presenters/psul_index_presenter_spec.rb (hunk 1 above)
Comment (banukutlu) [reply spec/presenters/psul_index_presenter_spec.rb]: 😄

**Pull request merged**